zanata-war/src/main/java/org/zanata/action/LanguageAction.java

```diff
@@ -128,6 +134,13 @@
 
     private List<SelectablePerson> searchResults;
 
+    private Map<LocaleRole, Boolean> roles = new HashMap<LocaleRole, Boolean>();
+    {
```

Comment:
I don't mind instance initialisers, but we should have a blank line to separate it from the Map declaration. This way looks a bit misleading.